docs/Third-party.md

```diff
@@ -44,92 +43,230 @@ _*You only need to install ITK if you plan to perform partial volume correction
 export FREESURFER_HOME="/Applications/freesurfer"
 source ${FREESURFER_HOME}/SetUpFreeSurfer.sh &> /dev/null
+```
+
+### FSL
+
+We recommend installing [**FSL 6.0**](https://fsl.fmrib.ox.ac.uk/).
+
+#### Installation
 
-# FSL
-# Uncomment the line below if you are on Mac:
-#export FSLDIR="/usr/local/fsl"
-# Uncomment the line below if you are on Linux:
-#export FSLDIR="/usr/share/fsl/6.0"
+##### On Linux
+
+Download it from [here](https://fsl.fmrib.ox.ac.uk/fsldownloads) and follow the instructions on the [FSL wiki](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FslInstallation/Linux).
+
+##### On MacOS
+
+Download it from [here](https://fsl.fmrib.ox.ac.uk/fsldownloads) and follow the instructions on the [FSL wiki](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FslInstallation/MacOsX).
+
+#### Configuration
+
+##### On Linux
+
+Make sure to have the following environment variables defined:
+
+```bash
+export FSLDIR="/usr/share/fsl/6.0"
 export PATH="${FSLDIR}/bin":${PATH}
 source ${FSLDIR}/etc/fslconf/fsl.sh
+```
 
-# Matlab
-export MATLAB_HOME="/path/to/your/matlab/bin/"
-export PATH=${MATLAB_HOME}:${PATH}
-export MATLABCMD="${MATLAB_HOME}/matlab"
+##### On MacOS
 
-# SPM
-export SPM_HOME="/path/to/your/spm12"
+Make sure to have the following environment variables defined:
+
+```bash
+export FSLDIR="/usr/local/fsl"
+export PATH="${FSLDIR}/bin":${PATH}
+source ${FSLDIR}/etc/fslconf/fsl.sh
 ```
 
-We recommend installing `ANTS >= 2.5.0` from which no environment variable are needed.
-Nonetheless, if you are using an older version of ANTS, make sure to have the following environment variables defined:
+### ITK
+
+You can find more details about `ITK` [here](https://itk.org/).
+
+#### Installation
+
+##### On Linux
+
+Follow the instructions on the [ITK blog](https://blog.kitware.com/itk-packages-in-linux-distributions/).
+
+##### On MacOS
+
+Follow the instructions on the [ITK blog](https://blog.kitware.com/kitware-packages-on-os-x-with-homebrew/).
+
+### MRtrix3
+
+You can find more details about `MRtrix3` [here](http://www.mrtrix.org).
+
+#### Installation
+
+You can find the official instructions on the `MRtrix` [website](https://www.mrtrix.org/download/).
+
+##### On Linux
+
+You have basically two options:
+
+- [Use the official conda package](https://www.mrtrix.org/download/linux-anaconda/).
+- [Build MRtrix3 from source](https://mrtrix.readthedocs.io/en/latest/installation/build_from_source.html#linux).
+
+!!! note
+    Note that using the conda package should be easier in most cases.
+
+##### On MacOS
+
+- [Use the official conda package](https://www.mrtrix.org/download/macos-anaconda/).
+- [Use the MacOS pre-compiled application package installer](https://www.mrtrix.org/download/macos-application/).
+- [Use the Homebrew formula](https://github.com/MRtrix3/homebrew-mrtrix3) (although large dependencies such as `XCode` and `Qt5` are required).
+
+!!! note
+    As for Linux, note that using the conda package should be easier in most cases.
+
+### Matlab
```

Comment:
Matlab is the only software that doesn't have an installation section. Is it ok?

Reply:
Good question.
It was already like this prior to this PR and I'm not sure why this is the case tbh... 🤔 
I think Matlab users will have a lot of ways to install it depending on specificities which are way outside of Clinica's scope (maybe your university or institution provides some licenses and custom ways to install it for example).
I think we can keep it like this.